packages/core/echo/echo-db/src/host/echo-host.ts

```diff
@@ -24,7 +24,7 @@ const INDEXER_CONFIG: IndexConfig = {
 
 export type EchoHostParams = {
   kv: LevelDB;
```

Comment:
not a great name. params probably a bit less bad?

Reply:
We're using `type <class name>Params` pattern a lot in the code. Also quite frequently saving params as a field of the class to avoid verbosity. Do you think we should change it?